packages/suite/src/constants/suite/device.ts

```diff
@@ -9,28 +9,33 @@ export const DEFAULT_STRENGTH: Record<DeviceModelInternal, number> = {
     [DeviceModelInternal.T1B1]: 256,
     [DeviceModelInternal.T2T1]: 128,
     [DeviceModelInternal.T2B1]: 128,
+    [DeviceModelInternal.T3T1]: 128,
 };
 
 export const MAX_ROWS_PER_PAGE: Record<DeviceModelInternal, number> = {
     [DeviceModelInternal.T1B1]: 4,
     [DeviceModelInternal.T2T1]: 5,
     [DeviceModelInternal.T2B1]: 4,
+    [DeviceModelInternal.T3T1]: 5,
```

Comment:
Is this somehow tested/confirmed by FW guys? Or is it just blind shoot? Do we know it?

Reply:
no, i have no idea what this means

Reply:
cc @matejcik

Reply:
from the name I'm guessing this is for address breaking? how many lines of text fit on a screen?
then that's for you to answer @Hannsek :)

Reply:
We won't change the max number of lines on a screen because of Suite. Suite won't be able to change the bahaviour so it would be broken. So the number of lines stays as for TT.